lib/user/getCollectibles.js

```diff
@@ -0,0 +1,55 @@
+// Includes
+const http = require('../util/http.js').func
+
+// Args
+exports.required = ['userId']
+exports.optional = ['assetType', 'sortOrder', 'limit']
+
+// Define
+function getCollectibles (userId, assetType, sortOrder, limit, pageCursor, currentCollectibles) {
+  return new Promise((resolve, reject) => {
+    const allowedLimits = [10, 25, 50, 100]
+
+    const httpOpt = {
+      url: `//inventory.roblox.com/v1/users/${userId}/assets/collectibles`,
+      options: {
+        qs: {
+          assetType: assetType,
+          sortOrder: sortOrder || 'Asc',
+          limit: limit <= 100 ? allowedLimits.reduce((prev, curr) => Math.abs(curr - limit) < Math.abs(prev - limit) ? curr : prev) : 100,
```

Comment:
What is going on here? Could you leave a comment explaining it?
Not just for my benefit, just for general maintainability.